isolate/initialdispatch.go

```diff
@@ -155,6 +188,19 @@ func (d *initialDispatch) Handle(id uint64, r *msgp.Reader) (Dispatcher, error)
 		}
 
 		return d.onSpawn(rawProfile, name, executable, args, env)
+	case workersMetrics:
+		if err = checkSize(_onMetricsArgsNum, r); err != nil {
+			log.G(d.ctx).Errorf("wrong args count for slot %d", id)
+			return nil, err
+		}
+
+		var uuids []string
+		if uuids, err = readStringsSlice(r); err != nil {
+			log.G(d.ctx).Errorf("wrong workersMetrics request framing: %v", err)
+			return nil, err
+		}
+
+		return d.onWorkersMetrics(uuids)
```

Comment:
are u sure that not
`return d.onWorkersMetrics(uuids), nil`
?

Reply:
I bet he's sure)